tests/Io/FileSystemTest.php

```diff
@@ -183,15 +310,15 @@ public function testReadSymbolicLink()
 
     public function testWrite()
     {
-        FileSystem::write('write', 'write content');
+        FileSystem::write('write.txt', 'write content');
 
-        $this->assertFileContains('write', 'write content');
+        $this->assertFileContains('write.txt', 'write content');
     }
 
     public function testRead()
     {
-        $this->file_put_contents('read', 'read content');
+        $this->file_put_contents('read.txt', 'read content');
 
-        $this->assertSame('read content', FileSystem::read('read'));
+        $this->assertSame('read content', FileSystem::read('read.txt'));
```

Comment:
Could you please remove all the `.txt` you added above? The add no value to the existing tests.